src/app/dynamic-forms/fields/rich-text/rich-text.component.ts

```diff
@@ -214,6 +253,13 @@ export class RichTextComponent implements OnDestroy, AfterViewInit {
       case TiptapMarkType.Code:
         this.editor?.commands.toggleCode();
 
+        break;
+      case TiptapMarkType.CodeBlock:
+        this.editor?.commands.toggleCodeBlock();
+
+        break;
+      case TiptapMarkType.Paragraph:
```

Comment:
Is there an explicit paragraph function on this? I thought paragraphs were added automatically to every line break.

Reply:
In the dropdown a 'Paragraph' command was added, so I needed this to revert a heading back to a paragraph.

Reply:
Fair enough!